lib/commands.js

```diff
@@ -17,18 +17,14 @@ var changeFunctionName = (function () {
     }
 }());
 
-// TODO: Rewrite this including the invidual commands into a Commands class
-// that provided a functionality to add new commands to the client
-
-commands.list.forEach(function (command) {
-
+var addCommand = function (command) {
     // Some rare Redis commands use special characters in their command name
     // Convert those to a underscore to prevent using invalid function names
     var commandName = command.replace(/(?:^([0-9])|[^a-zA-Z0-9_$])/g, '_$1');
 
     // Do not override existing functions
-    if (!RedisClient.prototype[command]) {
-        RedisClient.prototype[command.toUpperCase()] = RedisClient.prototype[command] = function () {
+    if (!RedisClient.prototype[commandName]) {
+        RedisClient.prototype[commandName.toUpperCase()] = RedisClient.prototype[commandName] = function () {
```

Comment:
This is a minimal changed behavior. The `command` is the true redis name and the `commandName` would fail in that case.